openqabot/aggrsync.py

```diff
@@ -22,11 +21,7 @@ def __init__(self, args: Namespace) -> None:
     def __call__(self) -> int:
         update_setting = []
         for product in self.product:
-            try:
-                update_setting += get_aggregate_settings_data(self.token, product)
-            except EmptySettings as e:
-                log.info(e)
-                continue
```

Comment:
Did you change the behavior of `get_aggregate_settings_data`? I didn't find that so I'm wondering whether the changed version is correct. (I wouldn't trust the fact the unit tests are still passing.)